pkgs/tools/misc/crypto-tracker/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "crypto-tracker";
+  version = "0.1.8";
+
+  src = fetchFromGitHub {
+    owner  = "Nox04";
+    repo   = "crypto-tracker";
+    rev    = "v${version}";
+    sha256 = "sha256-8tTaXpHZWcDq0Jfa9Hf258VYwfimLhYjCAzD4X/Ow0s=";
+  };
+
+  vendorSha256 = "sha256-ORdDrZ61u76mz2oZyxfdf7iuo9SnuQeDxESt9lORhgQ=";
+
+  meta = with lib; {
+    description = "This is a simple script written in Go to retrieve the latest price for several cryptocurrencies using the CoinMarketCap API";
```

Comment:
_Nitpick_: I'd personally condense it to: _Program to retrieve the latest price for several cryptocurrencies using the CoinMarketCap API_

Reply:
I agree with this recommendation.